sources/packages/backend/apps/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -212,6 +232,56 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return found.length === locations.length;
   }
 
+  /**
+   * Check if location code is already registered for the institution.
+   * @param locationCode location code.
+   * @param options object that should contain:
+   * - `locationId` location id in case it is an update of the location or
```

Comment:
I liked the way that the comments were applied to inline type:
![image](https://user-images.githubusercontent.com/61259237/204939033-9ba4f7fd-1fad-441a-b77f-cd3e7430b7ba.png)